backend/HitMeApp/HitMeApp.Users/Models/User.cs

```diff
@@ -3,6 +3,9 @@
     internal class User
     {
         public string Email { get; protected init; }
-        public string Name { get; protected init; }
+        public string Name
+        {
+            get; protected init;
```

Comment:
Why would this property be spanned across multiple lines? Few lines above we’ve got `Email` property with exactly the same signature which is written *normally*. Has automatic formatting done something wrong? From my perspective I’d prefer formatting applied to `Email`.

Reply:
Dude this is something that got spread across multiple lines by aplying rosely;s code rules to it. I have no other commeth to it except for this one,